File: swig/python/test_normalize_numexp.py
#coding:utf-8
import sys
from normalize_numexp import *

n = NormalizeNumexp("ja")
text = "魔女狩りは15世紀〜18世紀にかけてみられ、全ヨーロッパで4万人が処刑された"
result = StringVector(0)

n.normalize(text, result)
print "text:",text
for r in result :
    print r

File: swig/ruby/test-normalize-numexp.rb
#coding:utf-8
require "normalize_numexp"

n = Normalize_numexp::NormalizeNumexp::new("ja")
text = "魔女狩りは15世紀〜18世紀にかけてみられ、全ヨーロッパで4万人が処刑された"
result = Normalize_numexp::StringVector::new(0)

n.normalize(text, result)
print "text:#{text}\n"
result.each do |r|
		print "#{r}\n"
end
